src/lib_ccx/ccx_encoders_scc.c

```diff
@@ -0,0 +1,371 @@
+// Reference: https://www.govinfo.gov/app/details/CFR-2011-title47-vol1/CFR-2011-title47-vol1-sec15-119/summary
+// Implementatin issue: https://github.com/CCExtractor/ccextractor/issues/1120
+
+#include "lib_ccx.h"
+#include <stdbool.h>
+
+// Adds a parity bit if needed
+unsigned char odd_parity(const unsigned char byte)
+{
+	return byte | !(__builtin_parity(byte) % 2) << 7;
```

Comment:
This is GNU. Have you tried to build it with Visual Studio as well?

Reply:
Is there a function I should use instead or should I just add a function to replace this one?

Reply:
I haven't tried with Visual Studio though I don't expect it to work as I added it knowing it was an extension

Reply:
Take a look at 
int cc608_parity(unsigned int byte)

Reply:
Fixed